api/client/accesslist/accesslist.go

```diff
@@ -475,3 +475,22 @@ func (c *Client) GetSuggestedAccessLists(ctx context.Context, accessRequestID st
 
 	return accessLists, nil
 }
+
+// ListUserAccessLists returns a paginated list of all access lists where the
+// user is explicitly an owner or member
```

Comment:
Does `explicitly an owner or member` means that the nested ownership membership will not be listed / reflected by this endpoint ?